Documentation/security/policy/language.rst

```diff
@@ -1174,8 +1174,6 @@ for instructions.
 Deny Policies
 =============
 
-.. include:: ../../beta.rst
```

Comment:
Removing the beta status for a feature is not a negligible change that we should sneak in an otherwise unrelated commit. Can you please make a separate commit and give the motivations for the change?